src/main/java/ua/team3/carsharingservice/service/impl/PaymentHandlerForFine.java

```diff
@@ -21,4 +25,21 @@ public long calculateDays(Rental rental) {
     public BigDecimal calculateAmount(BigDecimal dailyFee, long rentalDays) {
         return FINE_MULTIPLAYER.multiply(dailyFee).multiply(BigDecimal.valueOf(rentalDays));
     }
+
+    @Override
+    public String formBillingDetails(Rental rental) {
```

Comment:
I think, it can even be in separate class. (Single Responsibility)

Reply:
I use this in the strategy of forming payment. It seems to me that this does not violate SRP.